apps/emqx_bridge_pulsar/src/emqx_bridge_pulsar_impl_producer.erl

```diff
@@ -70,10 +70,9 @@
 
 callback_mode() -> async_if_possible.
 
-%% there are no queries to be made to this bridge, so we say that
-%% buffer is supported so we don't spawn unused resource buffer
-%% workers.
-is_buffer_supported() -> true.
+%% consumer bridges don't need resource workers
+query_mode(_Config) ->
+    no_queries.
```

Comment:
pulsar is like Kafka, it has a buffer.

Reply:
should return `simple_async` here to keep it compatible as before this PR.

Reply:
Thanks, I think I was missled byt the comment that existed before my change.